modules/cli/src/integrationTest/java/org/apache/ignite/internal/cli/commands/CliCommandTestNotInitializedIntegrationBase.java

```diff
@@ -106,6 +107,10 @@ protected void execute(String... args) {
         exitCode = cmd.execute(args);
     }
 
+    protected CommandLine getCmd() {
```

Comment:
Why do you need this method? You can just make cmd protected.

Reply:
I wouldn't like to give an access to fields event for children. But I could rename the method to `cmd()`